Yoast/yoast-acf-analysis
# Fixing issue with too many calls to admin-ajax.

## Summary

This PR can be summarized in the following changelog entry:

* Seems that unnecessary calls where being made to query_attachments it was making calls even when the attachment ids were empty. 
* The issue was when the attachment_id value was getting fetched there was no check to see if it was empty before it was added into the array which made the attachment_ids never empty.  
* Added validation before attachment_id gets added to array.

## Test instructions

* Visit edit post page and make sure unnecessary calls are not made to the action query_attachments.

Fixes #

* Fix for #120
